src/views/CreateFamily.vue

```diff
@@ -88,13 +90,16 @@ export default class CreateFamily extends Mixins(ValidationMixin) {
   }
 
   get isFormInValidState() {
-    return (
-      (this.currentEmail === '' || this.isEmailValid(this.currentEmail)) &&
-      this.isDisplayNameValid(this.familyName)
-    );
+    return this.isDisplayNameValid(this.familyName);
+  }
+
+  get isEmailInValidState() {
+    return this.currentEmail !== '' && this.isEmailValid(this.currentEmail);
   }
 
   addEmail() {
+    if (!this.isEmailInValidState) return;
```

Comment:
Is this a safeguard? The add button wouldn't be shown if the email is invalid, so this method would never be called with invalid email.